src/Groups/GAPGroups.jl

```diff
@@ -874,5 +874,49 @@ the smallest integer $n$ such that `G` has a central series of length $n$.
 An exception is thrown if `G` is not nilpotent.
 """ nilpotency_class(G::GAPGroup)
 
-#
+@doc Markdown.doc"""
+    describe(G::GAPGroup)
+
+Return a string that describes some aspects of the structure of `G`.
+
+This works well if `G` is an abelian group or a finite simple group
+or a group in one of the following series:
+symmetric, dihedral, quasidihedral, generalized quaternion,
+general linear, special linear.
+
+For other groups, the function tries to decompose `G` as a direct product
+or a semidirect product,
+and if this is not possible as a non-splitting extension of
+a normal subgroup $N$ with the factor group `G`$/N$,
+where $N$ is the centre or the derived subgroup or the Frattini subgroup
+of `G`.
```

Comment:
The GAP manual contains a full description of the output "syntax" I believe. Maybe we can copY that, too? Could be done later though, in a future PR

Reply:
I have now added a markdown table with the notation.